build_tools/get_tag.py

```diff
@@ -29,6 +29,13 @@ def get_version_from_tag(tag):
         tag = os.getenv('BUILD_SOURCEBRANCH').split('/')[-1]
         f.write(get_version_from_tag(tag))
 
+elif os.getenv('GITHUB_REF') and \
+        os.getenv('GITHUB_REF').startswith('refs/tags/'):
```

Comment:
Does GH really use the `refs/tags` that Azure does? That's convenient.

Reply:
[They sure do!](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/using-environment-variables#default-environment-variables)